src/assets/src/components/queueEditors.tsx

```diff
@@ -93,14 +93,19 @@ export function ManageHostsEditor(props: ManageHostsEditorProps) {
     const hostsSoFar = props.hosts.map((host, key) => (
         <ListGroup.Item key={key}>
             <UserDisplay user={host} />
-            <div className='float-right'>
-                <RemoveButton
-                    onRemove={() => props.onRemoveHost(host)}
-                    size='sm'
-                    disabled={props.disabled || host.id === props.currentUser?.id}
-                    screenReaderLabel='Remove Host'
-                />
-            </div>
+            {
+                (host.id !== props.currentUser?.id)
+                    && (
+                        <div className='float-right'>
+                            <RemoveButton
+                                onRemove={() => props.onRemoveHost(host)}
+                                size='sm'
+                                disabled={props.disabled}
+                                screenReaderLabel='Remove Host'
+                            />
+                        </div>
+                    )
+            }
```

Comment:
@teshsopher, to address the contrast issues with the disabled "Remove Host" trash icon button, I decided to try simply removing that button when the host is the current user. To support this, I also made the phrase "You cannot remove yourself as a host" bold. What do you think about this solution? I wanted to avoid repeated language from the instructions, but I'm curious whether you think this works as part of your holistic vision for the UI.

Reply:
I think this works, based on the code, but I'd like to see it visually too

Reply:
@teshsopher, it looks like this:
<img width="500" alt="Screen Shot 2020-10-29 at 2 25 34 PM" src="https://user-images.githubusercontent.com/35741256/97616302-c1416300-19f2-11eb-802a-538aa9613281.png">